cogboard-webapp/src/components/PopoverWithControls.js

```diff
@@ -39,7 +46,20 @@ export const PopoverWithControls = ({
       >
         {withCopy ? <Button onClick={copyBody}>Copy</Button> : null}
         <Button onClick={handlePopoverClose}>Close</Button>
-        <StyledPopoverText>{body}</StyledPopoverText>
+        {bodyMessage === 'NO MATCH' ? (
+          <StyledPopoverTextWrapper>
+            <StyledPopoverText>
+              <StyledPopoverHeader>EXPECTED RESPONSE BODY</StyledPopoverHeader>
+              {expectedResponseBody}
+            </StyledPopoverText>
+            <StyledPopoverText>
+              <StyledPopoverHeader>RECIVED RESPONSE BODY</StyledPopoverHeader>
```

Comment:
I guess you wanted to write 'RECEIVED' here

Reply:
Fixed in commit 297f448